src/rpm/package.rs

```diff
@@ -34,6 +35,12 @@ pub struct RPMPackage {
 }
 
 impl RPMPackage {
+    pub fn open<P: AsRef<Path>>(path: P) -> Result<Self, RPMError> {
+        let rpm_file = std::fs::File::open(path.as_ref())?;
```

Comment:
Could we add `fs_err` to avoid the lacking io error Info? I'd be much in favor of that.

Reply:
Doc comment would be good 😊

Reply:
Are you referring to https://docs.rs/fs-err/latest/fs_err/?

Reply:
I think everything else should be `Reaser` and `Writer` trait bounded

Reply:
There's a variant of RPMError for it, I'm not sure what is using it.